### PR TITLE
Add an animation for HLD

### PR DESCRIPTION
**Pull request by brebenelmihnea**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [ ] I have tested my code.
- [ ] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [ ] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.

### CODE REVIEW

**Comment by brebenelmihnea** [comment]
Sure, but how do I add it ?

**Comment by SansPapyrus683** [comment]
why's the file in graph-traversal

**Comment by brebenelmihnea** [comment]
I think its' better to have all the animation in the same place

**Comment by SansPapyrus683** [comment]
ok but your animation for hld is in the directory labeled `graph-traversal`
i can't help but notice a discrepancy there
@ryanchou-dev what u think

**Comment by brebenelmihnea** [comment]
What if I rename `graph-traversal` to `animations` ? I will make more animations. I don't think it's best to make a directory for each topic's animations

**Comment by SansPapyrus683** [comment]
yeah

**Comment by brebenelmihnea** [comment]
@SansPapyrus683  Done. I renamed the directory and I chaged all the paths to animations accordingly

File: content/5_Plat/HLD.mdx [hunk 1]
```diff
@@ -79,6 +79,11 @@ $\mathcal{O}(\log N)$ light edges, computing the value on the path from node $x$
 to node $y$ will take $\mathcal{O}(\log^2 N + \log N)$ = $\mathcal{O}(\log^2 N)$
 time. We can answer $Q$ queries in $\mathcal{O}(Q \log^2 N)$ time.
 
+Here is an animations of how it works:
```
Comment (SansPapyrus683) [inline content/5_Plat/HLD.mdx:82]:
```suggestion
Here's a helpful animation of how the algorithm works:

```

**Review by SansPapyrus683 — APPROVED** [review]

File: content/5_Plat/HLD.mdx (hunk 1 above)
Comment (brebenelmihnea) [reply content/5_Plat/HLD.mdx]: Done

**Comment by brebenelmihnea** [comment]
@dustin-miao @dongliu0426 @andrewwangva  can you approve please ?

**Comment by brebenelmihnea** [comment]
@envyaims  @ryanchou-dev please review

**Review by dustin-miao — APPROVED** [review]
One small change

File: content/4_Gold/Tree_Euler.mdx [hunk 2]
```diff
@@ -424,7 +424,9 @@ size of $i$.
 
 Here's a small animation of the tour if you're still confused.
```
[the remark this answers is not in the document]
Comment (brebenelmihnea) [reply content/4_Gold/Tree_Euler.mdx]: Done.

**Review by dustin-miao — APPROVED** [review]

**Pull request merged**